hazelcast/util.py

```diff
@@ -83,7 +94,19 @@ def to_millis(seconds):
     :return: (int), result of the conversation in milliseconds.
     """
     if seconds >= 0:
-        return int(seconds * 1000)
+        return int(seconds * MILLISECONDS_IN_SECONDS)
+    return seconds
+
+
+def to_nanos(seconds):
```

Comment:
Here the parameter is always in seconds right? So in that case why not name the function as `seconds_to_nanos(seconds)`. Also this function does not do what it says. As a caller when I call this function I expect to have nano seconds as the return value however if the given seconds is negative then it doesnt return nanoseconds. This is confusing. Imagine someone calling this in the future without realizing that? It could result in bugs at best. If this is really the behavior you want to get then name the function like that. But I think this behavior is weird. Can you explain why are we returning the original negative value in the second case?

Reply:
I tried to mimic the behaviour of the old `to_millis` method without thinking too much about it. I think it's reasoning is that when you pass a negative time value it indicates the infinite or the default value in the context of Hazelcast time parameters (like TTL) so it is unnecessary to the multiplication here. But you are right this does not make sense. I will change them. For the naming, I think `to_millis` or `to_nanos` is just enough because time (and datetime) module returns everything as seconds. So, when you are doing a conversion to millis or nanos, it is almost always from seconds.